platform/mds/mds-performance-tests/src/test/java/org/motechproject/mds/performance/osgi/MdsDiskSpaceUsageIT.java

```diff
@@ -87,18 +87,27 @@ public void testEudeDiskSpaceUsage ()
         generator.generateDummyInstances(entityDto.getId(), INSTANCES);
 
         WebApplicationContext context = ServiceRetriever.getWebAppContext(bundleContext, MDS_BUNDLE_SYMBOLIC_NAME);
-        LocalPersistenceManagerFactoryBean localPersistenceManagerFactoryBean = context.getBean(LocalPersistenceManagerFactoryBean.class);
-        PersistenceManagerFactory persistenceManagerFactory = localPersistenceManagerFactoryBean.getObject();
 
-        JDOConnection con = persistenceManagerFactory.getPersistenceManager().getDataStoreConnection();
-        Connection nativeCon = (Connection) con.getNativeConnection();
+        LocalPersistenceManagerFactoryBean dataPersistenceManagerFactoryBean = (LocalPersistenceManagerFactoryBean) context.getBean("&dataPersistenceManagerFactoryBean");
```

Comment:
It is not obvious why you used '&' before bean name. However I noticed that this is a FACTORY_BEAN_PREFIX. Please change creation of this string using this constant from BeanFactory. It will be more clear to understand.